projects/angular/src/data/datagrid/_datagrid.clarity.scss

```diff
@@ -692,6 +692,15 @@
         display: none;
       }
     }
+    .datagrid-row-sticky {
+      .datagrid-column {
+        .datagrid-column-separator {
+          &::after {
+            background-color: tables-variables.$clr-table-border-color;
+          }
+        }
+      }
+    }
```

Comment:
This will create a color difference between the separator and "handler". Overall the handle is a separator too which is draggable. I suspect it should be the same color. 
Maybe check with design to confirm.

Reply:
This was the design provided by Lazar:
<img width="776" alt="Rows" src="https://github.com/vmware-clarity/ng-clarity/assets/1913702/48b9df74-8664-4576-8c1a-d57580806c89">

The active handlers have been updated to the new darker color, the inactive remain the same as per Lazar's comment below:
Inactive separators - object/--cds-alias-object-border-color and
Active handlers - interaction/--cds-alias-object-interaction-border-color (which makes the contrast 4,26:1).

Reply:
Yeah, I checked quickly with Grigor too. We have columns being resizable by default and there is no option to disable it so we wouldn't have mixed separators there with different colors. Thanks for clarifying!